app/ts/components/App.tsx

```diff
@@ -286,11 +290,9 @@ export function App() {
 				value = { inputValue.value } 
 				onInput = { e => handleInput(e.currentTarget.value) }
 			/>
-
+			
```

Comment:
Nitpick: I think this particular change is wrong and likely be undone in the next commit (next time VSCode auto-formats the file).  Usually this is fixed on save, not sure how you snuck it in.